src/screens/ChatScreen/AnonymousChatScreen.tsx

```diff
@@ -22,7 +22,7 @@ const {height} = Dimensions.get('window');
 export const styles = StyleSheet.create({
   keyboardAvoidingView: {
     flex: 1,
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.white` to `COLORS.almostBlack` for the `backgroundColor` property of `keyboardAvoidingView` style might affect readability and contrast on the screen. Please ensure that all text and elements within this view have been tested for accessibility and are clearly visible against the new background color.
```diff
-    backgroundColor: COLORS.white
+    backgroundColor: COLORS.almostBlack
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->